### PR TITLE
 Remove some fake kicker costs 

### PR DESCRIPTION
**Pull request by tool4ever**

### CODE REVIEW

**Comment by Hanmac** [comment]
@tool4ever checkout `K:Presence:Dragon` to be removed

**Comment by tool4ever** [comment]
> Does Dragon's Fire work for the AI after the change? Can't test atm, just remember that card being quite finicky for the AI and sensitive to implementation changes:)

It works just like before (=semi-optimally) because the logic for adding kicker is also closely related to any other OptionalCostValue

**Comment by tool4ever** [comment]
Tested text change and it worked fine

**Review by Agetian — APPROVED** [review]

File: forge-game/src/main/java/forge/game/cost/CostPartMana.java [hunk 1]
```diff
@@ -132,6 +132,7 @@ public ManaCost getManaCostFor(SpellAbility sa) {
             return sa.getPaidList(CostExile.HashLKIListKey, true).get(0).getManaCost();
         }
         if (isEnchantedCreatureCost() && sa.getHostCard().isEnchantingCard()) {
+            // TODO human can still activate on TDFC backside
             return sa.getHostCard().getEnchantingCard().getManaCost();
```
Comment (Hanmac) [inline forge-game/src/main/java/forge/game/cost/CostPartMana.java:136]: @tool4ever did you test what happen when it gets the "no cost" from a TDFC?

Comment (tool4ever) [reply forge-game/src/main/java/forge/game/cost/CostPartMana.java:136]: For some reason it will allow payment, will debug it more in another PR

**Review by Hanmac — APPROVED** [review]

**Pull request merged**